shardingsphere-sql-parser/shardingsphere-sql-parser-mysql/src/main/antlr4/imports/mysql/DALStatement.g4

```diff
@@ -40,7 +40,7 @@ showTableStatus
     ;
 
 showColumns
-    : SHOW EXTENDED? FULL? (COLUMNS | FIELDS) fromTable fromSchema? (showLike | showWhereClause_)?
+    : SHOW EXTENDED? FULL? (COLUMNS | FIELDS) fromTable fromSchema?  (showLike_ | showWhereClause_)?
```

Comment:
`showLike_` and `showLike` can make developer confuse.
It is better to rename `showLike_` to `showColumnLike_`.

Reply:
ok